bindings/python/py_src/tokenizers/trainers/__init__.pyi

```diff
@@ -136,6 +136,12 @@ class UnigramTrainer(Trainer):
             special_tokens: List[Union[str, AddedToken]]:
                 A list of special tokens the model should know of.
 
+            initial_alphabet: List[str]:
```

Comment:
Hmm I kind of think we should throw an error instead of silently taking the first char instead.  I understand it's to be coherent with other "initial_alphabets" but I think people don't read docs/docstrings as much as we would like, throwing an error would be more helpful to them.